pkgs/servers/matrix-synapse/matrix-appservice-irc/update.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nodePackages.node2nix nodejs-12_x curl jq nix-update
+
+set -euo pipefail
+# cd to the folder containing this script
+cd $(dirname $0)
```

Comment:
```suggestion
cd "$(dirname "$0")"
```

Reply:
You sure? Why don't the inner quotes need escaping?

Reply:
Not a shell magician, can tell you that shellcheck recommended that to me recently though :smile:

Reply:
> You sure? Why don't the inner quotes need escaping?

No, escaping is reset in the subshell.